### PR TITLE
[msal-angular] Update docs and samples

### PR DESCRIPTION
**Pull request by jo-arroyo**
This PR: 
- Updates the msal-angular v2 docs with information on how to handle EventError, in response to issue #2761
- Updates the `angular11-b2c-sample` with type checking 
- Updates `angular10-browser-sample` with correct request parameters from `@azure/msal-browser@2.8`, in response to issue #2763

### CODE REVIEW

File: lib/msal-angular/docs/v2-docs/events.md [hunk 1]
```diff
@@ -41,8 +41,50 @@ export class AppComponent implements OnInit, OnDestroy {
...
-For the full example of using events, please see the sample [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v2-samples/angular10-sample-app/src/app/app.component.ts#L29).
+For the full example of using events, please see our sample [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v2-samples/angular10-sample-app/src/app/app.component.ts#L29).
 
 ## Table of events
 
 For more information about the `EventMessage` object, including the full table of events currently emitted by `@azure/msal-browser` (including descriptions and related payloads), please see the documentation [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/events.md).
+
+## Handling errors with events
+
+As the `EventError` in `EventMessage` is defined as `AuthError | Error | null`, an error should be validated as the correct type before accessing specific properties on it.
```
Comment (jo-arroyo) [inline lib/msal-angular/docs/v2-docs/events.md:52]: I wonder if more should be said on the browser docs about this too? Would this be useful for msal-react? @jasonnutter @tnorling

Comment (jasonnutter) [reply lib/msal-angular/docs/v2-docs/events.md:52]: Yeah probably not a bad idea.

**Review by jasonnutter — APPROVED** [review]

File: lib/msal-angular/docs/v2-docs/events.md (hunk 1 above)
Comment (tnorling) [reply lib/msal-angular/docs/v2-docs/events.md:52]: Yes, I think a note in the events doc in browser would be useful. I'm going to put together an error handling doc for react soon so don't worry about that for now.

**Review by tnorling — APPROVED** [review]

File: lib/msal-angular/docs/v2-docs/events.md (hunk 1 above)
Comment (jo-arroyo) [reply lib/msal-angular/docs/v2-docs/events.md:52]: Makes sense. Browser doc updated.

**Pull request merged**